src/common/utils/setup-helpers.js

```diff
@@ -561,7 +561,9 @@ function supportLegacyOptions(options: Object): PartialKPOptionsObject {
     ['metadata.poster', 'sources.poster'],
     ['metadata', 'sources.metadata'],
     ['logLevel', 'log.level'],
-    ['ui.components.fullscreen.inBrowserFullscreenForIOS', 'playback.inBrowserFullscreen']
+    ['ui.components.fullscreen.inBrowserFullscreenForIOS', 'playback.inBrowserFullscreen'],
+    ['plugins.visibility.threshold', 'visibility.playerThreshold'],
+    ['plugins.visibility.floating', 'plugins.floating']
```

Comment:
`floating` will be directly under plugins?

Reply:
yes

Reply:
visibility has changed to floating?